diff --git a/lua/ui/icons.lua b/lua/ui/icons.lua
index ee0b09c..3aa2079 100644
--- a/lua/ui/icons.lua
+++ b/lua/ui/icons.lua
@@ -60,6 +60,11 @@ M.statusline_separators = {
       left = "",
       right = "",
    },
+
+   fire = {
+      left = " ",
+      right = "  ",
+   },
 }
 
 M.devicons = {
diff --git a/lua/ui/statusline/modules.lua b/lua/ui/statusline/modules.lua
index 3b1f517..915cae4 100644
--- a/lua/ui/statusline/modules.lua
+++ b/lua/ui/statusline/modules.lua
@@ -36,7 +36,7 @@ local M = {}
 
 M.mode = function()
    local m = vim.api.nvim_get_mode().mode
-   local current_mode = "%#" .. modes[m][2] .. "#" .. "  " .. modes[m][1]
+   local current_mode = "%#" .. modes[m][2] .. "#" .. "  " .. modes[m][1]
    local mode_sep1 = "%#" .. modes[m][2] .. "Sep" .. "#" .. sep_r
 
    return current_mode .. mode_sep1 .. "%#ST_EmptySpace#" .. sep_r
@@ -117,13 +117,13 @@ M.LSP_status = function()
 end
 
 M.cwd = function()
-   local dir_icon = "%#St_cwd_icon#" .. " "
+   local dir_icon = "%#St_cwd_icon#" .. "  "
    local dir_name = "%#St_cwd_text#" .. " " .. fn.fnamemodify(fn.getcwd(), ":t") .. " "
    return (vim.o.columns > 120 and ("%#St_cwd_sep#" .. sep_l .. dir_icon .. dir_name)) or ""
 end
 
 M.cursor_position = function()
-   local left_sep = "%#St_pos_sep#" .. sep_l .. "%#St_pos_icon#" .. " "
+   local left_sep = "%#St_pos_sep#" .. sep_l .. "%#St_pos_icon#" .. "  "
 
    local current_line = fn.line "."
    local total_line = fn.line "$"
